adds changelogs (#497)
matter-labs/zksync-web-era-docs: docs/api/hardhat/hardhat-zksync-verify.md

This task returns a verification id if the request was successfully sent.<br/>
You can use this id to check the status of your verification request as described in the section above.

> If the request was NOT sent successfully, the return value will be `-1`.

Reviewer: ```suggestion
If the request was NOT successful, the return value is `-1`.
```
